specification/hugr.md

```diff
@@ -424,7 +424,7 @@ output of each of these is a sum type, whose arity is the number of outgoing
 control edges; the remaining outputs are those that are passed to all
 succeeding nodes.
 
-The three nodes labelled "Const" are simply generating a predicate with one empty
+The three nodes labelled "Const" are simply generating a choice with one empty
```

Comment:
If a Choice (capitalized) is a type, is the corresponding value a choice (lowercase)? Or should "choice" here be "value of Choice type"? (the "one empty value to pass" might need changing too, not quite sure I understand that